pkg/antctl/raw/traceflow/command.go

```diff
@@ -249,13 +270,16 @@ func parseFlow() (*v1alpha1.Packet, error) {
 	}
 
 	pkt := new(v1alpha1.Packet)
+	// TODO: support IPv6.
```

Comment:
Lan Luo added IPv6 support in Antrea 1.0: https://github.com/vmware-tanzu/antrea/pull/1995

Reply:
Great to know. Removed to the TODO.